internal/content_update_policy/default_content_update_policy_resource.go

```diff
@@ -0,0 +1,713 @@
+package contentupdatepolicy
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/crowdstrike/gofalcon/falcon/client"
+	"github.com/crowdstrike/gofalcon/falcon/client/content_update_policies"
+	"github.com/crowdstrike/gofalcon/falcon/models"
+	"github.com/crowdstrike/terraform-provider-crowdstrike/internal/scopes"
+	"github.com/hashicorp/terraform-plugin-framework-validators/int64validator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/booldefault"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+// Ensure the implementation satisfies the expected interfaces.
+var (
+	_ resource.Resource                   = &defaultContentUpdatePolicyResource{}
+	_ resource.ResourceWithConfigure      = &defaultContentUpdatePolicyResource{}
+	_ resource.ResourceWithImportState    = &defaultContentUpdatePolicyResource{}
+	_ resource.ResourceWithValidateConfig = &defaultContentUpdatePolicyResource{}
+)
+
+// NewDefaultContentUpdatePolicyResource is a helper function to simplify the provider implementation.
+func NewDefaultContentUpdatePolicyResource() resource.Resource {
+	return &defaultContentUpdatePolicyResource{}
+}
+
+// defaultContentUpdatePolicyResource is the resource implementation.
+type defaultContentUpdatePolicyResource struct {
+	client *client.CrowdStrikeAPISpecification
+}
+
+// defaultContentUpdatePolicyResourceModel is the resource model.
+type defaultContentUpdatePolicyResourceModel struct {
+	ID                      types.String `tfsdk:"id"`
+	PlatformName            types.String `tfsdk:"platform_name"`
```

Comment:
PlatformName is not a valid field. The default content update policy works exactly like the content update policy resource. You shouldn't add fields.

Reply:
You did not remove PlatforName. Also default policies do not need to be imported first.

Reply:
default_sensor_update_policy is a different resource so it has different fields. PlatformName is not a valid field. 

default_content_update_policy will have the same fields as content_update_policy minus a few that do not exist like enabled. 

Remove PlatformName and use the